R/zzz.R

```diff
@@ -4,6 +4,13 @@
   if (is.null(getOption("rbabylon.bbi_exe_path"))) {
     options("rbabylon.bbi_exe_path" = "bbi")
   }
+
+  # set bbi minimum version
+  if (is.null(getOption("rbabylon.bbi_min_version"))) {
+    options("rbabylon.bbi_min_version" = package_version("2.1.0"))
```

Comment:
I think we're gonna pin this to 2.2.0 right? If so, this is gonna fail in drone until we "officially release" that, aka `https://www.github.com/metrumresearchgroup/babylon/releases/latest/download/bbi_linux_amd64.tar.gz` pulls version 2.2.0